### PR TITLE
feat: Added the first part of the series on security (supply chain)

### PR DESCRIPTION
**Pull request by ixchelruiz**

### CODE REVIEW

File: _posts/2024-04-30-security-slsa.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title: 'Building blocks'
+authors: [ 'ixchel' ]
+featuredImage: security 
+excerpt: "Security requires continuous effort, as there is no singular, straightforward solution that can be universally applied. The ever-changing nature of software means that what may be considered secure today could become vulnerable tomorrow"
+permalink: '/2024/04/30/security-slsa.html'
```
Comment (madmas) [inline _posts/2024-04-30-security-slsa.md:7]:
```suggestion
permalink: '/2024/06/12/security-slsa.html'
```

please also rename the file

File: _posts/2024-04-30-security-slsa.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+In the figure below, you can get a taste of _a few_ of the inflection points in the security of the entire software supply chain. SLSA focuses primarily on supply chain integrity and secondarily on availability. Integrity means protecting against tampering or unauthorised modification at any stage of the software lifecycle. SLSA distinguishes between source health and build health.
+
+**Source integrity:** Ensuring that the software producer's intent is reflected in all changes to the source code. An organisation's intent is difficult to define. Therefore, SLSA is expected to approximate this as approval by two authorised representatives.
+
+**Build integrity:** Ensuring that the package is built from the correct, unmodified sources and dependencies according to the build recipe defined by the software publisher. Ensuring that artefacts are not modified as they move between development stages.
+
+**Availability:** Ensure that all code and change history is available for investigation and incident response, and that the package can be built and maintained in the future. 
+
+![SLSA Supply chain threats](/assets/posts/2024-04-30-security-slsa/2024-04-30-supply-chain-threats.svg)
```
Comment (madmas) [inline _posts/2024-04-30-security-slsa.md:51]:
```suggestion
![SLSA Supply chain threats](/assets/posts/2024-06-12-security-slsa/2024-06-12-supply-chain-threats.svg)
```

please also rename directory and file

**Review by madmas — CHANGES_REQUESTED** [review]
great article, thank you

only chaged the publish date :)

File: _posts/2024-04-30-security-slsa.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+**Build Level 3 (Hardened builds)** provides additional protection against specific threats by ensuring that both the build and source platforms meet auditing standards and maintain the integrity of the provenance. The build platform implements strong controls to prevent runs from interfering with each other, even within the same project, and to prevent secret material used to sign the provenance from being accessible to custom build steps.
+
+Artifacts Integrity is about the ability to trust the authenticity of artifacts, meaning verifying that the artifact you get is really the original artifact uploaded by its author. Establishing the provenance of the artifacts produce is extremely important. So what is _"Provenance"_ all about?
```
Comment (grigala) [inline _posts/2024-04-30-security-slsa.md:66]:
```suggestion
Artifacts Integrity is about the ability to trust the authenticity of artifacts, meaning verifying that the artifact you get is really the original artifact uploaded by its author. Establishing the provenance of the artifacts produced is extremely important. So what is _"Provenance"_ all about?
```

File: assets/featuredImages/security.png
Comment (ixchelruiz) [inline assets/featuredImages/security.png:1]: @grigala Thank You!

**Review by grigala — APPROVED** [review]

**Pull request merged**